docs/workforce-management-framework/topics/pro-working-with-passportauth.adoc

```diff
@@ -1,51 +0,0 @@
-[id='pro-working-with-passportauth-{chapter}']
```

Comment:
This was just renamed to "[id='{context}-pro-passportauth-pointing-to-a-datasource']"